src/Extension/SandboxExtension.php

```diff
@@ -121,7 +121,9 @@ public function ensureToStringAllowed($obj, int $lineno = -1, ?Source $source =
     {
         if (\is_array($obj)) {
             foreach ($obj as $v) {
-                $this->ensureToStringAllowed($v, $lineno, $source);
+                if (\is_object($v) || \is_array($v)) {
```

Comment:
is_array still causes the issues in Drupal.

Reply:
is_object works.